tests/foreman/ui/test_discoveredhost.py

```diff
@@ -217,10 +150,21 @@ def test_positive_update_name(
 
 @pytest.mark.tier3
 @pytest.mark.upgrade
+@pytest.mark.on_premises_provisioning
+@pytest.mark.parametrize('module_provisioning_sat', ['discovery'], indirect=True)
+@pytest.mark.parametrize('pxe_loader', ['bios', 'uefi'], indirect=True)
+@pytest.mark.rhel_ver_match('9')
```

Comment:
why just 9?

Reply:
We just want to check it through the UI , no need for all the parametrization in my opinion